includes/core-testing-note-aspnet.md

```diff
@@ -0,0 +1,13 @@
+>[!IMPORTANT]
+>If you're _not_ using the **.NET Core 3.0 SDK**, you must add two additional package references, `Microsoft.AspNetCore.App` and `Microsoft.AspNetCore.Mvc.Testing`.
+>
+>```
+>dotnet add package Microsoft.AspNetCore.App
```

Comment:
This should be added without a package version. Does `dotnet add package` respect that? I imagine you'd get a version and then get a warning in the build.

Reply:
It adds whatever version is the latest that it uses. I can't see an easy way to force it to versionless apart from editing the csproj file directly.

Reply:
cc @dsplaisted. We should be careful recommending this, then. 

I forget what happens when you have a version, does it win or just warn?

Reply:
It should warn

Reply:
warn and then ignore or warn and then use what you say?